lms/djangoapps/courseware/field_overrides.py

```diff
@@ -14,6 +14,7 @@
 `LmsFieldData`.  This means overrides will be in effect for all scopes covered
 by `authored_data`, e.g. course content and settings stored in Mongo.
 """
+# pylint: disable=missing-docstring
```

Comment:
Rather than a global disable, it would be better to document which functions you think don't need docstrings with the disable in-line.

Also, I think that generally, having docstrings that can be turned into documentation (via Sphinx) is a win. The function signature gives you information, but usually (even well written), it doesn't give enough details about the expected behavior and edge-cases.

Comments, I tend to agree with, since they'll only be seen by someone who is looking at the code in question. But docstrings should be usable to avoid *needing* to look at the code itself.